proto/cdcpb.proto

```diff
@@ -94,7 +118,17 @@ message ChangeDataRequest {
     bytes end_key = 6;
     // Used for CDC to identify events corresponding to different requests.
     uint64 request_id = 7;
+
     kvrpcpb.ExtraOp extra_op = 8;
+
+    oneof request {
+        // A normal request that trying to register change data feed on a region.
+        Register register = 9;
```

Comment:
Why do we need it?

Reply:
I think there may be more different types of request so I use `oneof` field in the request. The `register` represents the original request that registers changefeed to a region.